migration_tools.drush.inc

```diff
@@ -248,6 +263,51 @@ function drush_migration_tools_mt_migrate_file_folders($organization) {
 }
 
 /**
+ * Drush command callback for migration-tools-html-file-list.
+ *
+ */
+function drush_migration_tools_html_file_list($directory) {
+  // Set up options
+  $format = drush_get_option('format');
+  $outfile = drush_get_option('outfile');
+  $mask = '/\.htm/';
+
+  switch ($format) {
+    case "csv":
+      $delimiter = ", ";
+      $newline = "\n";
+      break;
+    default:
+      $delimiter = "\t";
+      $newline = "\n";
+  }
+
+  // Fetch file list
+  $options = array();
+  $files = file_scan_directory($directory, $mask, $options);
+
+  // Generate output
+  $output = "Path" . $delimiter . "File Name" . $delimiter . $newline;
+  foreach ($files as $file) {
+    // We only want the path to the file, exclusive of the file name, so we drop the last part of the uri
+    $fullpath = explode("/", $file->uri, -1);
```

Comment:
We want the filenames.  The purpose is to get a full list of all html files.   We can collapse subdirectories by hand if it turns out they have 100 of all the same kind of pages.  However, to start we want all of them.